docker/import-osm/import_osm.sh

```diff
@@ -25,7 +25,8 @@ function import_pbf() {
         -cachedir "$IMPOSM_CACHE_DIR" \
         -read "$pbf_file" \
         -deployproduction \
-        -write $diff_flag
+        -write $diff_flag \
```

Comment:
what if `$diff_flag` or `$CONFIG_JSON` are not defined?  Wouldn't that break all existing users?  You might want to use this variant instead:
```suggestion
        ${diff_flag:+-write "$diff_flag"} \
```

Reply:
It is computed here. But I does not touch this part except the trailing `\`.

https://github.com/openmaptiles/openmaptiles-tools/blob/9068be14e38e942a8b180b868c66e6d34cb8fb9a/docker/import-osm/import_osm.sh#L13-L18

Reply:
sorry, my mistake about the -write flag, if it was there before, no need to change that